packages/ui-causal-graph-editor/docs/changelog.md

```diff
@@ -2,6 +2,10 @@
 title: Changelog
 ---
 
+## NEXT
+
+-   Updated the type of `additionalLegends` to also allow defining nodes
```

Comment:
Let's add a changelog entry for the new `defaultLegends` prop

Reply:
Added good catch